x-pack/plugins/canvas/public/state/reducers/resolved_args.js

```diff
@@ -130,6 +131,9 @@ export const resolvedArgsReducer = handleActions(
         return state;
       }, transientState);
     },
+    [setWorkpad]: (transientState, {}) => {
+      return set(transientState, 'resolvedArgs', {});
```

Comment:
nit: you don't *need* `set` here, `return { ...transientState, resolvedArgs: {} };` would do. But it's already being used here, so there's no real issue using it some more.